mingw_conf.sh

```diff
@@ -66,8 +98,8 @@ set(CMAKE_SYSTEM_PROCESSOR x86_64)
 
 # We'll need to use posix threads in order to use
 # C++11 features, such as std::thread.
-set(CMAKE_C_COMPILER \${TOOLCHAIN_PREFIX}-gcc${mingwPosix})
-set(CMAKE_CXX_COMPILER \${TOOLCHAIN_PREFIX}-g++${mingwPosix})
+set(CMAKE_C_COMPILER ${MINGW_CC})
```

Comment:
`TOOLCHAIN_PREFIX` and `MINGW_BASE` seem to be the same thing and used interchangeably.  Perhaps the former could be replaced by the latter entirely?